web/modules/custom/dpl_webmaster/src/Form/InstallOrUpdateModule.php

```diff
@@ -291,4 +291,42 @@ public function submitForm(array &$form, FormStateInterface $form_state): void {
     }
   }
 
+  /**
+   * Copy uploaded module into place and run updates.
+   *
+   * @param string[] $projects
+   *   List of projects to update.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   Form state to set redirect on.
+   */
+  protected function redirectToUpdates(array $projects, FormStateInterface $form_state): void {
+    // Most of this has been lifted from UpdateReady::submitForm().
+    drupal_get_updaters();
+
+    $updates = [];
+    $directory = _update_manager_extract_directory();
+
+    $project_real_location = NULL;
+    foreach ($projects as $project) {
+      $project_location = $directory . '/' . $project;
+      $updater = Updater::factory($project_location, $this->root);
+      $project_real_location = $this->fileSystem->realpath($project_location);
+      $updates[] = [
+        'project' => $project,
+        'updater_name' => get_class($updater),
+        'local_url' => $project_real_location,
+      ];
+    }
+
+    // Contrary to UpdateReady::submitForm(), we don't check file owners or
+    // support FTP method.
+    $this->moduleHandler->loadInclude('update', 'inc', 'update.authorize');
+    $filetransfer = new Local($this->root, $this->fileSystem);
+    // @phpstan-ignore function.notFound (It's loaded by the loadInclude)
+    $response = update_authorize_run_update($filetransfer, $updates);
+    if ($response instanceof Response) {
```

Comment:
Please consider adding the file to [`scanFiles` in `phpstan.neon.dist`](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/9d306e2a7595a5bb9edfac27c291e65ea2e94e1e/phpstan.neon.dist#L74-L74) instead of using an ignore.

This is what we have done in similar situations.